src/Pages/Main/Components/Nav/Nav.scss

```
@@ -0,0 +1,239 @@
+@import '../../../../Styles/variables.scss';
+
+a {
+  text-decoration: none;
+  color: $black;
+}
+
+.Nav {
+  .mainContainer {
+    display: flex;
+    justify-content: center;
+    height: 75px;
+    border-bottom: 1px solid $lightGrey;
+  }
+
+  .subContainer {
+    display: flex;
+    align-items: center;
+  }
+
+  // left
+  .navBarLeftContainer {
+    display: flex;
+  }
+
+  .navBarLeft {
+    display: flex;
+    align-items: center;
+
+    li {
+      margin: 0px 15px 0px 15px;
+    }
+  }
+
+  .navBarHover:hover {
+    padding: 28px 0px 28px 0px;
+    border-bottom: 2px solid $black;
+    .navDropdown {
+      display: block;
+    }
+  }
+
+  .dropdownContainer {
+    display: flex;
+    justify-content: space-evenly;
+    position: absolute;
+    padding-top: 50px;
+    margin-top: 30px;
+    left: 0px;
+    width: 100%;
+    height: 500px;
+    border: black solid;
+    background-color: $lightGrey;
+  }
+
+  .navDropdown {
+    // display: none;
+    display: block;
+    z-index: 1;
+  }
+
+  .allItems {
+    .allItemsTitle {
+      font-weight: bold;
+    }
+  }
+
+  .elecItems {
+    .elecItemsTitle {
+      font-weight: bold;
+      padding-bottom: 15px;
+    }
+
+    li {
+      padding: 15px 0px 15px 0px;
+      text-align: left;
+    }
+  }
+
+  .wearItems {
+    .wearItemsTitle {
+      font-weight: bold;
+      padding-bottom: 15px;
+    }
+
+    li {
+      padding: 15px 0px 15px 0px;
+      text-align: left;
+    }
+  }
+
+  .livingItems {
+    .livingItemsTitle {
+      font-weight: bold;
+      padding-bottom: 15px;
+    }
+
+    li {
+      padding: 15px 0px 15px 0px;
```

```suggestion
    li {
      padding: 15px 0px;
```
padding 속성은 단축 속성 사용해서 같은 내용을 짧게 줄여줄 수 있습니다~ 비슷한 다른 부분에도 찾아서 적용해주세요!

---

감사합니다! 단축 속성 사용해서 같은 내용들 짧게 줄여줬습니다.